src/clarity/forms/_forms.clarity.scss

```diff
@@ -19,6 +18,8 @@ $clr-form-field-disabled-color: $clr-gray !default;
 $clr-checkbox-radio-height: $clr-icon-dimension-sm !default;
 $clr-form-icon-dim: $clr-icon-dimension-sm !default;
 $clr-custom-checkbox-radio-top: ($clr_baselineRem_1 - $clr-checkbox-radio-height) / 2 !default;
+$clr-checkbox-height: ($clr-checkbox-radio-height / 4 + 1);
```

Comment:
Is this the height of the checkbox or is it just the height of the hypen/dash symbol in the checkbox in the indeterminate state? If it is the second one, we should change the variable name to reflect that. Same for the width on the next line.